Update client.lua

Mesrine67 opened the pull request:
add lib.addKeybind()

## Description

<!-- What does your pull request change? Why should it be merged? Does it fix an issue? -->

## Checklist

<!-- Put an x inside the [ ] to check an item, like so: [x] -->

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My pull request fits the contribution guidelines & code conventions.

Mesrine67 committed:
Update client.lua

add lib.addKeybind()

Manason commented on qbx_cruise/client.lua:
```diff
@@ -69,16 +69,15 @@ local function TriggerCruiseControl()
-            TriggerCruiseControl()
-        else
-            exports.qbx_core:Notify(locale('error.cruise_control_unavailable'), 'error')
+local keybindCruiseControl = lib.addKeybind({name = 'toggle_cruise_control', description = 'Activer/Désactiver le régulateur de vitesse', defaultKey = 'Y',
```
make a locale

Manason commented on qbx_cruise/client.lua:
```diff
@@ -69,16 +69,15 @@ local function TriggerCruiseControl()
-            exports.qbx_core:Notify(locale('error.cruise_control_unavailable'), 'error')
+local keybindCruiseControl = lib.addKeybind({name = 'toggle_cruise_control', description = 'Activer/Désactiver le régulateur de vitesse', defaultKey = 'Y',
+    onPressed = function(self)
+        local vehicleClass = GetVehicleClass(cache.vehicle)
+        if cache.seat == -1 then
```
swap these two lines

Mesrine67 committed:
Update client.lua

Add locale Keybind cruise controle

Mesrine67 committed:
Update fr.json

Mesrine67 committed:
Update en.json

Mesrine67 reviewed (commented):
Up locale

Mesrine67 committed:
Update client.lua

Add return keybindCruiseControl 
exemple: 
```lua
local keybindCruiseControl = lib.load('@qbx_smallresources.qbx_cruise').keybindCruiseControl

if QBX.PlayerData.metadata.ishandcuffed then
    keybindCruiseControl:disable(true)
end
```
